### PR TITLE
blog: path validation vulnerability blog

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/website please take a look, would like to land soon.

File: locale/en/blog/vulnerability/september-2017-path-validation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Contact and future updates
+
+The current Node.js security policy can be found at https://nodejs.org/en/security/.
+
+Please contact security@nodejs.org if you wish to report a vulnerability in Node.js.
+
+Subscribe to the low-volume announcement-only nodejs-sec mailing list at
+https://groups.google.com/forum/#!forum/nodejs-sec to stay up to date
+on security vulnerabilities and security-related releases of Node.js
+ th Validation Vunerabilityand the projects maintained in the nodejs GitHub organisation.
```
Comment (lpinca) [inline locale/en/blog/vulnerability/september-2017-path-validation.md]: s/th Validation Vunerability//

**Comment by lpinca** [comment]
It might make sense to also update this https://github.com/nodejs/nodejs.org/blob/267cd83c73b6b03aa1b09012f192a85158e1ea6e/build.js#L287 accordingly.

**Comment by mhdawson** [comment]
Pushed changes to address comments

**Comment by mhdawson** [comment]
@lpinca thanks for catching those issues.

File: locale/en/blog/vulnerability/september-2017-path-validation.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+date: 2017-09-27T17:15:00.617Z
+category: vulnerability
+title: Path validation vulnerability, September 2017
+slug: sep-2017-path-validation
+layout: blog-post.hbs
+author: Michael Dawson
+---
+# Path Validation Vunerability
```
Comment (silverwind) [inline locale/en/blog/vulnerability/september-2017-path-validation.md]: `Vulnerability`

**Review by silverwind — APPROVED** [review]
LGTM with typo fixed

File: locale/en/blog/vulnerability/september-2017-path-validation.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+The Node.js project released a new versions of 8.x this week which incorporates
```
Comment (Trott) [inline locale/en/blog/vulnerability/september-2017-path-validation.md]: `new versions` -> `new version`

**Review by lpinca — APPROVED** [review]
LGTM with typos fixed.

**Review by cjihrig — APPROVED** [review]
LGTM with typos addressed.

**Comment by mhdawson** [comment]
pushed fixes, will land

**Comment by mhdawson** [comment]
Landed as ba715268e3b628451a3e5f6af202552c81b5b6f5

**Comment by refack** [comment]
As a postscript, since there are two people "implicated" with the issue this post is in reference to:
image
IMHO it would have been courteous to @-mention them during this review and before this went public.
Not a big deal, but like they say "finding out by reading about it in the paper is not the best feeling in the world"

**Comment by mhdawson** [comment]
@refack will keep that in mind.  It might make sense to PR some guidance into: https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md

**Pull request closed without merging**